setup.py

```diff
@@ -40,10 +40,11 @@
     author='Google Cloud Platform',
     author_email='googleapis-publisher@google.com',
     long_description=README,
-    namespace_packages=['google'],
     scripts=[],
     url='https://github.com/googleapis/google-resumable-media-python',
-    packages=setuptools.find_packages(exclude=('tests*',)),
+    packages=setuptools.find_namespace_packages(
+        exclude=("tests*")
```

Comment:
Thanks @parthea for adding this! Do we need to exclude the docs* files here as well?